nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/util/OSUtil.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.bootstrap.util;
+
+import java.lang.reflect.Field;
+
+import org.slf4j.Logger;
+import com.sun.jna.Pointer;
+import com.sun.jna.platform.win32.Kernel32;
+import com.sun.jna.platform.win32.WinNT;
+
+/**
+ * OS specific utilities with generic method interfaces
+ */
+public final class OSUtil {
```

Comment:
If I am reading the JNA license correctly, it looks like versions after 4.0 are ASLv2, and possibly dual licensed (lgpl and ASLv2). Per https://www.apache.org/legal/resolved#mutually-exclusive , it would appear just ASLv2 would need to be noted. And since libffi is included in JNA, and is MIT style, I would expect it would need to be called out as well. If you look at nifi-assembly, there are already many dependencies called out and listed if you want examples to reference